spec/rubocop/cop/layout/redundant_line_break_spec.rb

```diff
@@ -271,6 +271,19 @@ def resolve_inheritance_from_gems(hash)
             m(7 + 8 + 9)
         RUBY
       end
+
+      it 'properly corrects a method chain on multiple lines' do
```

Comment:
Since the newly added `gsub` has a regexp that doesn't include a line break, it's a bit more error prone than the already existing ones. So if you change the test case to
```ruby
      it 'properly corrects a method chain on multiple lines' do
        expect_offense(<<~RUBY)
          foo(' .x')
          ^^^^^^^^^^ Redundant line break detected.
            .bar
            .baz
        RUBY

        expect_correction(<<~RUBY)
          foo(' .x').bar.baz
        RUBY
      end
```
it fails because whitespace is removed by autocorrect inside the string.

Reply:
Oh good catch! Thank you, I’ll fix it up!

Reply:
@jonas054 I fixed that up and added more tests cases. Thank you!